pkgs/data/themes/adwaita-qt/default.nix

```diff
@@ -18,11 +28,13 @@ mkDerivation rec {
 
   buildInputs = [
     qtbase
+    qt5.qtx11extras
+    xorg.libxcb
```

Comment:
Instead, you would use a conditional here:

```suggestion
  ] ++ lib.optionals stdenv.isLinux [
    xorg.libxcb
```

Reply:
Thanks, applied